essentials-api/src/main/kotlin/io/github/nyliummc/essentials/api/builders/Text.kt

```diff
@@ -29,7 +29,7 @@ import net.minecraft.text.ClickEvent
 import net.minecraft.text.HoverEvent
 import net.minecraft.util.Formatting
 import java.util.function.Consumer
-import net.minecraft.text.Text as MCText
+import net.minecraft.text.BaseText as MCText
```

Comment:
Is there a reason we have to upcast to base text?

Reply:
Text itself doesn't have styled or setStyle, so this is required.

Reply:
Wouldn't MutableText suffice then?